08_voting/index.tex

```diff
@@ -1 +1,101 @@
+
+\textbf{Instant Runoff} is a voting method which is used in the elections for a single seat which consist of more than two candidates. 
+Individual voters rank their preference of candidates. The candidate who gets the least number of first-placed is eliminated, and this step is repeated until only one candidate remains.
+
+\textbf{Approval} is a voting method for a single winner in which each voter selects a subset of candidates. Approval will be used for the elections of roles where each voter picks a group of candidates, and the most named or approved candidate wins.
+
+In case of a \textbf{tied election}, there are three possible rules that can be used to eliminate one or more candidates who get the same number of votes:
+\begin{itemize}
+    \item The Condorcet method is one of the voting methods which selects the candidate that dominates other candidates in term of majority relation which ranks the candidates according to one-to-one comparisons. In other words, the Condorcet winner is the candidate that wins a majority in every one-to-one comparison against other candidates. However, Condorcet method is not guaranteed to work when preference lists form a cycle, which is known as the Condorcet paradox or voting paradox.
+    \item Hare Rule can be used in a tiebreak where all candidates with the least first-place votes will be all eliminated.
+    \item Coombs Rule could be used in place of Hare Rule if the iterative process of removing candidates with the least first-place votes is not preferred. Coombs Rule indicates that the candidate with the most last-placed votes will be removed.
+\end{itemize}
+
+In this coursework, the pairwise Condorcet winner is chosen for as a tiebreaker which states that a candidate who dominates the other candidate in term of majority relation wins.
+
+\section{Voting Protocol}
+\label{sec:VotingProtocol}
+The suggested voting protocol is adapted from an example of formal specification given in Robert's Rules of Order (RONR) that defines a series of sequential actions when a voting event happens.
+
+This protocol is applicable to any voting method at a certain state of the game.
+
+The sequence of actions of the voting protocol in this game when a voting event happens is as follows: The President selects a rule from the rule proposal list to be voted on; the President notifies the Speaker to hold a voting event for the selected rule; the Speaker calls for a vote for the selected rule by opening ballots to all islands to cast their votes; the Speaker closes the ballots after receiving votes from all islands; the Speaker counts the votes by calling an applicable voting method function for winner selection to produce a result; finally, the Speaker publicly declares the voting result whether the selected rule can be carried out or not. Figure~\ref{fig:RONRVotingProtocol} shows the sequence of the voting protocol for changing and/or adapting a new rule.
```

Comment:
This is an incredibly long sentence with lots of clauses linked by semicolons. It would be far clearer imo if you used a list here to separate this out for the reader and show that it is a series of steps/sequence of actions

Reply:
hm.. maybe we can do `\begin{itemize} .. \end{itemize}` or smt like that

Reply:
changed